### ISSUE TITLE
[Bug] PojavLauncher crashing on Air 2 iOS 15.7.5 Palera1n

### ISSUE THREAD

**Issue by Piipperi** [PojavLauncherTeam/PojavLauncher_iOS#577]
### Describe the bug

- Opening a jar from ”Install .jar” menu causes a nearly instant crash.
- Tried rd-132211, 1.5.2, 1.8.9, 1.19.4. All except 1.19.4 crash as soon as I move the mouse. 1.19.4 works for 20 - 30 seconds after loading the menu, before crashing.
- Played around with different renderers and RAM allocation options, no change.

### Steps To Reproduce

```markdown
1. Start PojavLauncher
2. Launch 1.8.9
3. Move the mouse (I used an external USB one)
```

### Expected Behavior

I expect the game to not crash

### Platform

```markdown
- Device model: iPad Air 2
- iOS version: iOS 15.7.5
- PojavLauncher version: 2.2
```

**Comment by khanhduytran0** [comment]
The log section is missing. Send your logs.

**Comment by Piipperi** [comment]
1.8.9 crash: [latestlog.old.txt](https://github.com/PojavLauncherTeam/PojavLauncher_iOS/files/11672885/latestlog.old.txt)
Opening OF installer crash: [latestlog.old.txt](https://github.com/PojavLauncherTeam/PojavLauncher_iOS/files/11672890/latestlog.old.txt)

**Comment by khanhduytran0** [comment]
Installer crash might’ve been fixed in 3.0, not sure about 1.8.9 since there is no indicate of crash in log, try checking Analytics Data
